### PR TITLE
Fix panics when integration is waiting for platform

### PR DESCRIPTION
**Pull request by astefanutti**
**Release Note**
```release-note
Fix panics when integration is waiting for platform
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by squakez — APPROVED** [review]

File: pkg/trait/openapi.go [hunk 1]
```diff
@@ -270,7 +270,7 @@ func (t *openAPITrait) createNewOpenAPIConfigMap(e *Environment, resource v1.Dat
 		return err
 	}
 	mc.GlobalSettings = data
-	//nolint: staticcheck,nolintlint
+	// nolint: staticcheck,nolintlint
```
Comment (tadayosi) [inline pkg/trait/openapi.go:273]: Btw, with upcoming golangci-lint version nolintlint should invalidate it if it's in the form of `// nolint:`. The correct one should be `//nolint:`.

**Review by tadayosi — APPROVED** [review]

**Comment by tadayosi** [comment]
Most CI checks are failing. Is it related to the fix?

File: pkg/trait/openapi.go (hunk 1 above)
Comment (astefanutti) [reply pkg/trait/openapi.go]: Reverted. My IDE keeps updating it!

**Comment by astefanutti** [comment]
> Most CI checks are failing. Is it related to the fix?

I've updated the unit tests to use the required platform mock.

**Pull request merged**